linux/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.14-stretch
+
+RUN apt-get -y update \
+    && apt-get -y install \
+        gettext-base \
+    && apt-get -y clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
FYI I had just added this to the other dockerfile to use `envsubst`

Reply:
I don't have a dockerfile anymore